### DISCUSSION TITLE
Show active preset on climate entity

### DISCUSSION THREAD

**Discussion by CFenner** [home-assistant/frontend#19037] [Dashboard]
The climate entity displays the active preset in the device view:
<img width="331" alt="Bildschirmfoto 2023-12-13 um 11 19 04" src="https://github.com/home-assistant/frontend/assets/9592452/f82010bc-f447-4dc4-9acf-e3da3e750bd1">

This is missing in the tile cards and in the climate card. One can add this with a feature, but that is not always sufficient. 

In ViCare integration, the heating has presets that are controlled by a schedule and are not allowed to set manual. In this case it would be nice to see the current active preset without being able to select/activate the one.

### Describe the behavior you expected

I would like to see the active preset somewhere, be it on the feature (non selectable option) or on the entity itself.

**Comment by karwosts** [comment]
This seems like a feature-request, not a bug with the existing implementation. 

As such, I will move this to discussions, as is appropriate for feature requests.

**Reply by CFenner** [reply]
I think this was visible with the *old* climate card though.
